helm-charts/bytebase/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: bytebase
-version: 1.1.0
+version: 1.1.1
```

Comment:
Change version to 1.1.1 and run `scripts/package_helm.sh`, Should we keep the old version(1.1.0) `tgz package` and `index.yaml` ?

Reply:
Yes! We should keep the old tgz package because artifact hub references them.